inference-engine/tests/functional/plugin/shared/src/subgraph_tests/concat_qunatization.cpp

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <memory>
+#include <tuple>
+#include <vector>
+#include <string>
+
+#include <ie_core.hpp>
+
+#include "common_test_utils/common_utils.hpp"
+#include "functional_test_utils/plugin_cache.hpp"
+
+#include "subgraph_tests/concat_quantization.hpp"
+
+
+namespace LayerTestsDefinitions {
+
+std::string ConcatQuantization::getTestCaseName(testing::TestParamInfo<LayerTestsUtils::basicParamsWithConfig> obj) {
+    InferenceEngine::Precision netPrecision;
+    InferenceEngine::SizeVector inputShapes, newInputShapes;
+    std::string targetDevice;
+    std::map<std::string, std::string> configuration;
+    std::tie(netPrecision, inputShapes, targetDevice, configuration) = obj.param;
+
+    std::ostringstream result;
+    result << "IS=" << CommonTestUtils::vec2str(inputShapes) << "_";
+    result << "netPRC=" << netPrecision.name() << "_";
+    result << "targetDevice=" << targetDevice;
+    for (auto const& configItem : configuration) {
+        result << "_configItem=" << configItem.first << "_" << configItem.second;
+    }
+
+    return result.str();
+}
+
+void ConcatQuantization::SetUp() {
+    std::vector<size_t> inputShape;
+    InferenceEngine::Precision netPrecision;
+    std::tie(netPrecision, inputShape, targetDevice, configuration) = this->GetParam();
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+
+    auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
+
+    std::vector<size_t> outFormShapes1 = { 1, 5, 32 };
```

Comment:
If internal shapes are fixed is there any sense to make network input shape as a parameter?